resource_activity/models/resource_activity_registration.py

```diff
@@ -138,15 +138,25 @@ def _compute_need_push(self):
                 registration.need_push = True
 
     @api.multi
-    @api.depends("allocations.state")
+    @api.depends(
+        "allocations",
+        "allocations.state",
+        "resources_available.allocation_id",
```

Comment:
Same here, if the allocation changes, Odoo considers the state changes. Might be worth checking though.

Reply:
indeed, you’re right. i made some tests and only the deepest field is needed.